gRPC-Core.podspec

```diff
@@ -178,6 +178,7 @@ Pod::Spec.new do |s|
     ss.libraries = 'z'
     ss.dependency "#{s.name}/Interface", version
     ss.dependency 'BoringSSL-GRPC', '0.0.21'
+    ss.dependency 'Libuv-gRPC'
```

Comment:
Would be nice if we could specify the version to ensure we don't end up pulling an older version in the case if we make a recent libuv version bump but forgot to push to cocoapod.